base/workflow/inst/batch_runs.R

```diff
@@ -0,0 +1,431 @@
+#!/usr/bin/env Rscript
+
+# if (!requireNamespace("huxtable")) {
+#   message("Installing missing package 'huxtable'")
+#   install.packages("huxtable")
+# }
+# 
+# if (!requireNamespace("htmlTable")) {
+#   message("Installing missing package 'htmlTable'")
+#   install.packages("htmlTable")
+# }
+
+## This script contains the Following parts:
+## Part 1 - Write Main Function
+## A. takes in a list defining specifications for a single run and assigns them to objects
+## B. The function then writes out a pecan settings list object
+## C. A directory is created from the ouput folder defined in a users config.php file
+## D. Settings object gets written into pecan.xml file and put in output directory.
+## E. workflow.R is copied into this directory and executed.
+## F. console output is stored in a file called workflow.Rout
+## Part 2 - Write run settings
+## A. Set BETY connection object. Needs user to define path to their pecan directory.
+## B. Find Machine ID
+## C. Find Models ids based on machine
+## D. Manually Define Met, right now CRUNCEP and AMerifluxlbl
+## E. Manually Define start and end date
+## F. Manually Define Output var
+## G. Manually Define Sensitivity and enesmble
+## H. Find sites not associated with any inputs(meaning data needs to be downloaded), part of ameriflux network, and have data for start-end year range.
+## Available ameriflux data is found by parsing the notes section of the sites table where ameriflux sites have a year range.
+## Part 3 - Create Run table
+## A. Create table that contains combinations of models,met_name,site_id, startdate, enddate, pecan_path,out.var, ensemble, ens_size, sensitivity args that the function above will use to do runs.
+## Part 4 - Run the Function across the table
+## A. Use the pmap function to apply the function across each row of arguments and append a pass or fail outcome column to the original table of runs
+## Part 5 - (In progress) Turn output table into a table
+
+#' .. title/description ..
+#'
+#' @param model_id Model ID (from `models` table)
+#' @param met Meteorology input source (e.g. "CRUNCEP")
+#' @param site_id Site ID (from `sites` table)
+#' @param start_date Run start date
+#' @param end_date Run end date
+#' @param pecan_path Path to PEcAn source code. Default is current
+#'   working directory.
+#' @param user_id
+#' @param output_folder
+#' @param dbfiles_folder
+#' @param pft_name
+#' @param ensemble_size
+#' @param ensemble_variable
+#' @param sensitivity
+#' @param db_bety_username
+#' @param db_bety_password
+#' @param db_bety_hostname
+#' @param db_bety_driver
+#' @return 
+#' @author Alexey Shiklomanov
+create_execute_test_xml <- function(model_id,
```

Comment:
Isn't there a function of the same name inside the package that's part of this same PR? Can we decide which we like better (or merge the two) so that there's only one version.